src/Datadog.Trace/Tracer.cs

```diff
@@ -675,6 +664,33 @@ private static IDogStatsd CreateDogStatsdClient(TracerSettings settings, string
             }
         }
 
+        private static ITraceWriter CreateTraceWriter(TracerSettings settings, IDogStatsd statsd)
+        {
+            IMetrics metrics = statsd != null
+                ? new DogStatsdMetrics(statsd)
+                : new NullMetrics();
+
+            switch (settings.Exporter)
+            {
+                case ExporterType.Zipkin:
+                    return new ExporterWriter(new ZipkinExporter(settings.AgentUri), metrics);
+                case ExporterType.Jaeger:
+                    return new ExporterWriter(new JaegerExporter(CreateJagerOptions(settings)), metrics);
+                default:
+                    return new AgentWriter(new Api(settings.AgentUri, TransportStrategy.Get(settings), statsd), metrics, maxBufferSize: settings.TraceBufferSize);
+            }
+        }
+
+        private static JaegerOptions CreateJagerOptions(TracerSettings settings)
```

Comment:
Just a minor typo, should be `CreateJaegerOptions`.

Reply:
Good catch! Fixed.